site/content/en/docs/tasks/setup_multikueue.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Setup a MultiKueue environment"
+date: 2024-02-26
+weight: 9
+description: >
+  Additional steps needed to setup the multikueue clusters.
+---
+
+Check the concepts section for a [MultiKueue overview](/docs/concepts/multikueue/). 
+
+## Worker Cluster
+
+When MultiKueue dispatches a workload from the manager cluster to a worker cluster, it expects that the job's namespace and LocalQueue also exist in the worker cluster.
+In other words, you should ensure that the worker cluster configuration mirrors the one of the manager cluster in terms of namespaces and LocalQueues.
+
+To create the sample queue setup in the `default` namespace apply:
+
+{{< include "examples/admin/single-clusterqueue-setup.yaml" "yaml" >}}
+
+### MultiKueue Specific Kubeconfig
+
+In order to delegate the jobs in a worker cluster, the manager cluster needs to be able to create, delete, and watch workloads and their parent Jobs.
+
+While `kubectl` is set up to use the worker cluster, download: 
```

Comment:
Maybe you can start the document by saying:
Let's assume that your manager cluster is named X and your worker clusters are named Y and Z. To follow this tutorial, ensure that the credentials for all these clusters should present in the kubeconfig in your local machine.

Then you can just say, run `kubectl config use-cluster worker-cluster-1` below.

Reply:
done